hs/GridResponse.hs

```diff
@@ -33,22 +33,22 @@ conflictDialog :: H.Html
 conflictDialog = createTag H.div "dialog" "" "Conflicting courses are difficult to manage. Make sure you understand the added responsibility of having two or more conflicting courses."
 
 coursePanel :: H.Html
-coursePanel = createTag H.div "course-select-wrapper" "col-md-2 col-xs-6" $ do
-                            createTag H.ul "course-select" "trapScroll-enabled" $ do
-                                   createTag H.li "clear-all" "" $ do
+coursePanel = createTag H.div "course-select-wrapper" "col-md-2 col-xs-6" $
+                            createTag H.ul "course-select" "trapScroll-enabled" $
```

Comment:
Sure does look that way, but it's spaces! I'll do something about the indentation, though.